azure-pipelines.yaml

```diff
@@ -3,6 +3,8 @@ variables:
   buildConfiguration: Release
   majorVersion: 0.0.0
   isReleaseTriggered: $[startsWith(variables['Build.SourceBranch'], 'refs/heads/action/release')]
+  isMasterTriggered: $[startsWith(variables['Build.SourceBranch'], 'refs/heads/master')]
+  Codeql.Enabled: true
```

Comment:
In the previous iterations this was found to be failing for all other languages other than dotnet. Is it still happening?

Reply:
Java will be handled with the workflow